Jenkinsfile

```diff
@@ -827,8 +881,12 @@ pipeline {
                     }
                     steps {
                         dir("${env.WORKSPACE}/linux-armel"){
-                            unstash "Stanc3Setup"
-                            sh "bash -x scripts/build_multiarch_stanc3.sh armel"
+                            cleanCheckout()
+                            sh """
+                                eval \$(opam env)
+                                dune subst
```

Comment:
I don't think this will work, because it's outside the building docker instance. Could we do it in `scripts/build_multiarch_stanc3.sh`? Just another thing in the `&&` chain in that file.